.github/workflows/ci-pr-target.yaml

```diff
@@ -0,0 +1,14 @@
+# This workflow supports jobs which needs the on.pull_request_target event
+# instead of the usual on.pull_request event.
+name: CI
```

Comment:
Good digging into the problem :+1: 

Since we are forced to have 2 wf files, we should probably rename them, otherwise is gets hard to identify them from the Actions tab.

Before we had one big wf for "CI" in general. Now we two, we can be a bit more specific, especially about this one. Suggestion: `name: Pkg coverage` (as the incoming last wf will also need the same PR commenting rights)

Reply:
Changed in https://github.com/canonical/chisel-releases/pull/199/commits/a4e618255f8e5363d9e9cbe212a20717ef95a874.